modules.json

```diff
@@ -40,25 +40,25 @@
                 "git_sha": "e937c7950af70930d1f34bb961403d9d2aa81c7d"
             },
             "rseqc/bamstat": {
-                "git_sha": "e937c7950af70930d1f34bb961403d9d2aa81c7d"
+                "git_sha": "28b023e6f4d0d2745406d9dc6e38006882804e67"
```

Comment:
Did you update this file manually? You will also need to update [this file](https://github.com/nf-core/rnaseq/blob/8094c42add6dcdf69ce54dfdec957789c37ae903/.nf-core.yml#L10-L16) with that commit. I added these modules to the blacklist for `nf-core modules update` because v4.0 of RSeqC wasnt' working when I last tried. If you use the latest version of nf-core/tools in the future to update the module i.e. `nf-core modules update` then it should take care of all of this and notify you that the module is blacklisted :)

Reply:
Thanks for the blacklist hint. I tried to use the `modules update` functionality, but failed as it told me it is up to date (as expected). Hence I used the `modules install --force --sha XXX <module name>`. 

To get this going I changed also the `.nf-core.yml` manually for now.

But I also tried your new way, but still failed. Not sure if it is a bug (I can report this if wanted) or a problem on my side. I currently use `nf-core/tool=2.1`.
I basically did:
1. checkout rnaseq-dev branch
2. change sha hash in `.nf-core.yml` with sed
3. run `nf-core modules update <module/name>`.

The log:
```
(nextflow_base) (128)[13:32:00]mertes@ouga08:~/projects/nf-core$ git clone --branch dev git@github.com:nf-core/rnaseq rnaseq-dev
Cloning into 'rnaseq-dev'...
(nextflow_base) (0)[13:32:15]mertes@ouga08:~/projects/nf-core$ cd rnaseq-dev/
(nextflow_base) (0)[13:32:26]mertes@ouga08:~/projects/nf-core/rnaseq-dev$ git status
On branch dev
Your branch is up to date with 'origin/dev'.

nothing to commit, working tree clean
(nextflow_base) (0)[13:32:40]mertes@ouga08:~/projects/nf-core/rnaseq-dev$ sed 's/5dd049047d01e72c01a519422f17e203bca343ac/28b023e6f4d0d2745406d9dc6e38006882804e67/' -i .nf-core.yml 
(nextflow_base) (0)[13:33:08]mertes@ouga08:~/projects/nf-core/rnaseq-dev$ git diff
diff --git a/.nf-core.yml b/.nf-core.yml
index 7b62557..74bf7fc 100644
--- a/.nf-core.yml
+++ b/.nf-core.yml
@@ -10,7 +10,7 @@ update:
     rseqc/bamstat: "e937c7950af70930d1f34bb961403d9d2aa81c7d"
     rseqc/inferexperiment: "e937c7950af70930d1f34bb961403d9d2aa81c7d"
     rseqc/innerdistance: "e937c7950af70930d1f34bb961403d9d2aa81c7d"
-    rseqc/junctionannotation: "5dd049047d01e72c01a519422f17e203bca343ac"
+    rseqc/junctionannotation: "28b023e6f4d0d2745406d9dc6e38006882804e67"
     rseqc/junctionsaturation: "e937c7950af70930d1f34bb961403d9d2aa81c7d"
     rseqc/readdistribution: "e937c7950af70930d1f34bb961403d9d2aa81c7d"
     rseqc/readduplication: "e937c7950af70930d1f34bb961403d9d2aa81c7d"
(nextflow_base) (130)[13:33:23]mertes@ouga08:~/projects/nf-core/rnaseq-dev$ nf-core modules update rseqc/junctionannotation

                                          ,--./,-.
          ___     __   __   __   ___     /,-._.--~\
    |\ | |__  __ /  ` /  \ |__) |__         }  {
    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                          `._,._,'

    nf-core/tools version 2.1



INFO     Updating 'nf-core/modules/rseqc/junctionannotation'                                                                                                                                                                                                                 update.py:228
ERROR    Could not fetch nf-core/modules file: 403                                                                                                                                                                                                                  modules_command.py:266
          https://api.github.com/repos/nf-core/modules/contents/modules/rseqc/junctionannotation/functions.nf?ref=28b023e6f4d0d2745406d9dc6e38006882804e67                                                                                                                                
```
Maybe a problem: `functions.nf` was not changed in this commit. But you still ask for the specific file with the given commit hash.